lib/reek/smells/attribute.rb

```diff
@@ -1,4 +1,6 @@
+require_relative 'smell_configuration'
```

Comment:
(Note that these are just my current ideas, I have tested them only to some extent in rather small personal projects; I’m not sure whether they’re feasible in larger codebases.)